<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectorClass</class>
 <widget class="QMainWindow" name="ProjectorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projector</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>左上侧长度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="protopleftLength">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000000</number>
           </property>
           <property name="value">
            <number>1580</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>右上侧长度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="protoprightLength">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000000</number>
           </property>
           <property name="value">
            <number>1570</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>水平投影宽度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="prohorWidth">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000000</number>
           </property>
           <property name="value">
            <number>760</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>左下侧长度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="probottomleftLength">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000000</number>
           </property>
           <property name="value">
            <number>1440</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>绕Z旋转</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="rotate">
           <property name="minimum">
            <number>-89</number>
           </property>
           <property name="maximum">
            <number>89</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>垂直投影宽度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="proverWidth">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000000</number>
           </property>
           <property name="value">
            <number>572</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="plane">
            <attribute name="title">
             <string>平面</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>左上侧长度</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="topleftLength">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>10000000</number>
                   </property>
                   <property name="value">
                    <number>1500</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>左下侧长度</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="bottomleftLength">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>10000000</number>
                   </property>
                   <property name="value">
                    <number>1380</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>右下侧长度</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="bottomrightLength">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>10000000</number>
                   </property>
                   <property name="value">
                    <number>1870</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="cylinder">
            <attribute name="title">
             <string>柱面</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="ball">
            <attribute name="title">
             <string>球面</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>504</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="projector.qrc"/>
 </resources>
 <connections/>
</ui>
